.circleci/config.yml

```diff
@@ -29,31 +80,22 @@ jobs:
             pip install black
             black --check sphinx/ tests/ notice-check.py
 
-      - run:
-          name: Make build directory
-          command: mkdir build
-
-      - run:
-          name: CMake
-          command: 'cmake -GNinja -DVIRTUAL_ONLY=ON ..'
-          working_directory: build
-
-      - run:
-          name: Ninja
-          command: ninja -j4
-          working_directory: build
-
-      - run:
-          name: Run tests
-          working_directory: build
-          command: ./tests.sh -VV --timeout 240 --no-compress-output -T Test
+  build_coverage:
+    executor: large_1804
+    steps:
+      - build_and_tests
+      - gather_and_push_coverage
 
-      - run:
-          name: Extract test results
-          working_directory: build
-          command: |
-            mkdir -p test-results/junit
-            xsltproc --stringparam suiteName "Virtual Build" ../tests/infra/ctest_to_junit.xslt Testing/*/Test.xml > test-results/junit/results.xml
+  build_san:
+    executor: large_1804
+    steps:
+      - build_and_tests:
+          cmake_flags: "-DSAN=ON"
+          suite_name: "SAN"
 
-      - store_test_results:
-          path: build/test-results
+workflows:
+  ci:
+    jobs:
+      - static_checks
```

Comment:
Does this gate the other two jobs?

Reply:
No, I started out that way, but it slows down the job too much. There ways around that we can explore, but for now I am comfortable with this because Circle is very good at canceling outdated jobs (unlike Pipelines), so this doesn't cause problems.